### ISSUE TITLE
PartByValueStatistics skip index by mistake

### ISSUE THREAD

**Issue by linhong-intel** [Intel-bigdata/OAP#285]
Test case. After disabled PartByValueStatistics, case passed.
```
test("index integrity test") {
    val data : Seq[(Int, String)] =
      (2, "this is test 1")::
        (6, "this is test 2")::
        (5, "this is test 3")::
        (7, "this is test 4")::
        (8, "this is test 5")::
        (3, "this is test 6")::
        (8, "this is test 7")::
        (9, "this is test 8")::
        (6, "this is test 9")::
        (9, "this is test 10")::
        (1, "this is test 11")::
        (2, "this is test 12")::
        (0, "this is test 13")::
        (8, "this is test 14")::
        (1, "this is test 15")::
        (1, "this is test 16")::
        (3, "this is test 17")::
        (9, "this is test 18")::
        (1, "this is test 19")::
        (7, "this is test 20"):: Nil

    data.toDF("key", "value").createOrReplaceTempView("t")
    sql("insert overwrite table oap_test select * from t")
    sql("create sindex index1 on oap_test (a) using bitmap") // Neither btree nor bitmap works

    val dfwithIdx = sql("SELECT * FROM oap_test WHERE a = 8")
    dfwithIdx.show
    val count1 = dfwithIdx.count
    sql("drop sindex index1 on oap_test")
    val count2 = sql("SELECT * FROM oap_test WHERE a = 8").count

    if (count1 != count2) {
      sql("SELECT * FROM oap_test").show
    }
    assert(count1 == count2)
  }
```

**Comment by linhong-intel** [comment]
We can disable PartByValueStatistics as a workaround if there is no time to fix

**Comment by linhong-intel** [comment]
Fixed by #286
